znai-docs/znai/release-notes/1.50/add-2021-06-15-python-function.md

```diff
@@ -0,0 +1 @@
+* Add [include-python](snippets/python#function-content) to include function's content
```

Comment:
@jyeakel @evinaik FYI, that's how I am trying to manage release notes and avoid merge conflicts at the same time.
any new PR with fix/feature creates a new file with what was added and then `include-markdowns` plugin renders the directory of them sorted by file names descending